app/jobs/send_user_invitation_job.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class SendUserInvitationJob < ActiveJob::Base
```

Comment:
```suggestion
class SendUserInvitationJob < ApplicationJob
```

Reply:
Should I move the job under `workers` folder too?

Reply:
yup : 👍